edit-post/assets/stylesheets/_z-index.scss

```diff
@@ -33,9 +33,6 @@ $z-layers: (
 	// Active pill button
 	".components-button.is-button {:focus or .is-primary}": 1,
 
-	// Should have lower index than anything else positioned inside the block containers
-	".editor-block-list__block-draggable": 0,
```

Comment:
Dragging no longer needs a dedicated DOM node. This PR removes the `editor-block-list__block-draggable` node from  `BlockDraggable`.